pkg/nsx/services/vpc/vpc.go

```diff
@@ -470,3 +521,232 @@ func (s *VPCService) Cleanup() error {
 
 	return nil
 }
+
+func (service *VPCService) needUpdateRule(rule *model.Rule, externalCIDR []string) bool {
+	des := rule.DestinationGroups
+	currentDesSet := sets.Set[string]{}
```

Comment:
is there possibility the destinationGroups contain duplicate IPBlocks? Why convert to Set?